solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -620,10 +620,15 @@ protected Map<String, Object> startSolr(
     if (!isWindows && cwdPath.length() > 1 && solrHome.startsWith(cwdPath))
       solrHome = solrHome.substring(cwdPath.length() + 1);
 
+    final var syspropArg =
```

Comment:
If you are adding this, should the lib directive be removed from the example configSet?

Reply:
It's already gone, afaict.  @epugh removed it (unrelated to this change) in [a recent PR here](https://github.com/apache/solr/pull/2861/files#diff-1b6e36f0bb5a2f3801d561bd2fa32cb1e354e8342f6cf80e6e6dbfa9ed54d83c)